### PR TITLE
Websites: Adding metadata to support the isPreview feature

### PR DESCRIPTION
**Pull request by panchagnula**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by derekbekoe** [comment]
@panchagnula Change the description.
Also, you don't need the last comma after DEPENDENCIES

**Comment by derekbekoe** [comment]
Also, it's not `azext_aem`

**Comment by derekbekoe** [comment]
@panchagnula Please take a look at this PR otherwise I'll close soon. Thanks.

**Comment by panchagnula** [comment]
@derekbekoe i will get to this soon - sorry was out of office & now working on //BUILD work items.

**Comment by panchagnula** [comment]
@derekbekoe  can you help me figure out the Travis CI build error?

File: src/index.json [hunk 1]
```diff
@@ -281,10 +281,11 @@
...
         "webapp": [
             {
-                "filename": "webapp-0.2.0-py2.py3-none-any.whl",
-                "sha256Digest": "a13151c899eabb66c3d853e9e46bcd5752f78e5a75c35eb09590f4673ac49279",
-                "downloadUrl": "https://github.com/panchagnula/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.0-py2.py3-none-any.whl",
+                "filename": "webapp-0.2.1-py2.py3-none-any.whl",
+                "sha256Digest": "fe3b28cc9cb2272a36c395ed6279b2a9f925b16877962bf2448a3fa56819bfa4",
+                "downloadUrl": "https://github.com/panchagnula/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.1-py2.py3-none-any.whl",
                 "metadata": {
+                    "azext.minCliCoreVersion": "2.0.24",
```
Comment (derekbekoe) [inline src/index.json]: You need to add `"azext.isPreview": true` here also.

**Pull request merged**